locales/fa/validation-inline.php

```diff
@@ -103,11 +103,11 @@
     'present'              => 'این مقدار باید وارد شده باشد.',
     'prohibited'           => 'این فیلد ممنوع است.',
     'prohibited_if'        => 'این فیلد زمانی که :other برابر با :value باشد ممنوع است.',
-    'prohibited_unless'    => 'This field is prohibited unless :other is in :values.',
-    'prohibits'            => 'This field prohibits :other from being present.',
+    'prohibited_unless'    => 'فیلد :attribute ممنوع است مگر اینکه مقدار :other در :values باشد.',
+    'prohibits'            => 'فیلد :attribute اجازه حضور فیلد :other را نمی دهد.',
     'regex'                => 'این فرمت نامعتبر است.',
     'required'             => 'این مقدار ضروری است.',
-    'required_array_keys'  => 'This field must contain entries for: :values.',
+    'required_array_keys'  => 'فیلد :attribute باید حاوی ورودی های :values باشد.',
```

Comment:
In an inline file, you do not need to access the attribute. The "this field..." option is used.